devito/finite_difference.py

```diff
@@ -137,18 +137,23 @@ def first_derivative(*args, **kwargs):
     dim = kwargs.get('dim', x)
     diff = kwargs.get('diff', h)
     order = kwargs.get('order', 1)
-    side = kwargs.get('side', 1)
+    side = kwargs.get('side', 0)
```

Comment:
no need for string comparison. As @vincepandolfo was saying, I was just suggesting to have something like ``right`` or ``left`` instead of ``1`` or ``-1``, or perhaps better words. This would improve the code